api/src/main/java/pl/kolendateam/dadcard/characterCard/entity/Character.java

```diff
@@ -199,14 +204,14 @@ public void calculateSkillPoints(int skPoints) {
         this.skillPoints += skPoints+abilitys.bonusIntelligence(abilitys);
     }
 
-    public void buySkills(int idSkill, int skPoints) {
+    public void buySkills(int idSkill, double skPoints) {
```

Comment:
skil point moze nie byc liczba calkowita?

Reply:
to zmieniłem w innym commicie. Więc, tu tego nie zmienie, bo i tak będzie używany inny DTO do tego